src/main/java/com/exasol/projectkeeper/BrokenLinkReplacer.java

```diff
@@ -0,0 +1,61 @@
+package com.exasol.projectkeeper;
+
+import java.util.*;
+
+import com.exasol.errorreporting.ExaError;
+
+/**
+ * Some maven projects define invalid / outdated links to their project homepage. Since PK writes theses links to the
+ * dependencies.md file each project again contains these broken links. Since that's not desirable and breaks the CI
+ * link checker, this class replaces the broken links by working ones.
+ * <p>
+ * This class has a built in mapping for some common broken links but also allows users to define new link mappings in
+ * the pom file.
+ * </p>
+ */
+public class BrokenLinkReplacer {
```

Comment:
What's the difference between BrokenLinkReplacer and DependenciesBrokenLinkReplacer?

Reply:
BrokenLinkReplacer in multi purpose just replacing links. 

DependenciesBrokenLinkReplacer replaces links in dependencies